Hack code_freeze to test new release notes automation

mokagio opened the pull request.

mokagio committed:
Hack `code_freeze` to test new release notes automation

dangermattic commented:
<!--
  0 Errors
  0 Warnings
  1 Message: This PR is still a Draft: some...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

mokagio committed:
Add dummy entry in changelog to test automation

mokagio committed:
Remove extra empty line from `RELEASE-NOTES.txt`

Otherwise, the automation will generate sources for the localization
with that additional empty which is confusing.

See:

- https://github.com/Automattic/simplenote-ios/pull/1648/commits/ca20c0d4c0ab1f945d3a9ee030f3a16e7846eb31
- https://github.com/Automattic/simplenote-ios/pull/1648/commits/28cd12c9b5cc68bdf553894de74afed893910b5c

This ought to be fixed at the release-toolkit level.

mokagio committed:
Update draft release notes for 4.53.

mokagio committed:
Delete release notes source and localization before code freeze

mokagio committed:
Update metadata strings

mokagio committed:
Release Notes: add new section for next version (4.54)
